yarn-project/simulator/src/public/execution.ts

```diff
@@ -18,8 +18,8 @@ import { type Gas } from '../avm/avm_gas.js';
  * The public function execution result.
  */
 export interface PublicExecutionResult {
-  /** The execution that triggered this result. */
-  execution: PublicExecution;
+  /** The execution request that triggered this result. */
+  executionRequest: PublicExecutionRequest;
```

Comment:
HA! I suggested it some comments above. I think it's ok, but I don't remember what this object has inside. I seem to recall there was some form of callRequest but I might be wrong. If yes, it might be a bit strange (but I'm not against it).

Note: I see below that 
```
PublicExecutionRequest = Pick<
  PublicCallRequest,
  'contractAddress' | 'functionSelector' | 'callContext' | 'args'
>;
```

I think the rename is fine.

Reply:
Yeah, it _is_ a call request.... Or part of one :sweat_smile: